### ISSUE TITLE
Crash RemoteServiceException

### ISSUE THREAD

**Issue by asynnestvedt** [nextcloud/android#7374]
### Steps to reproduce
1. Open photos
2. Tap on photo
3. Delete photo

### Expected behaviour
- Tell us what should happen

### Actual behaviour
- Tell us what happens

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
************ CAUSE OF ERROR ************

android.app.RemoteServiceException: Context.startForegroundService() did not then call Service.startForeground(): ServiceRecord{ee9e3a6 u0 com.nextcloud.client/.media.PlayerService}
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1973)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:224)
	at android.app.ActivityThread.main(ActivityThread.java:7560)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:539)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30140090
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: Xiaomi
Device: raphael
Model: Mi 9T Pro
Id: QKQ1.190825.002
Product: raphael_eea

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: V12.0.3.0.QFKEUXM

```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by RubenFixit** [comment]
I'm running into the same issue after deleting any file. These issue appears to already be captured by #6905 .

**Comment by aqaqsubin** [comment]
I think this issue is discussed #2010 
And seems to be fixed in #7340
